outputs.tf

```diff
@@ -14,38 +14,17 @@
  * limitations under the License.
  */
 
-output "dataset_id" {
-  value       = google_bigquery_dataset.main.id
-  description = "Unique id for the dataset being provisioned"
+output "bigquery_dataset" {
+  value       = google_bigquery_dataset.main
+  description = "Bigquery dataset resource."
 }
 
-output "dataset_name" {
-  value       = google_bigquery_dataset.main.friendly_name
-  description = "Friendly name for the dataset being provisioned"
+output "bigquery_tables" {
+  value       = { for table in var.tables : table["table_id"] => google_bigquery_table.main[table["table_id"]] }
+  description = "Map of bigquery table resources being provisioned."
 }
 
-output "dataset_project" {
-  value       = google_bigquery_dataset.main.project
-  description = "Project where the dataset and table are created"
+output "bigquery_tables_list" {
```

Comment:
Is there a compelling use case to output this as a map and as a list? I'm in favour of the map output; I'm just not sure providing it as a list as well provides any additional value.

Reply:
The decision was made based on some other modules having that approach, I don't know a real example, output has been deleted (once we have a usecase we can add back)